doc/src/base/scopedvalues.md

```diff
@@ -0,0 +1,86 @@
+# Scoped Values
+
+Scoped values provide an implementation of dynamical scoping in Julia.
+In particular dynamical scopes are propagated through [Task](@ref)s.
+
+!!! compat "Julia 1.11"
+    Scoped values were introduced in Julia 1.11. In Julia 1.7+ a compatible
+    implementation is available from the package ScopedValues.jl.
+
+In its simplest form you can create a [`ScopedValue`](@ref) with a
+default value and then use [`scoped`](@ref) to enter a new dynamical
+scope. The new scope will inherit all values from the parent scope 
+(and recursivly from all outer scopes) with the the provided scoped
+value taking priority over previous definitions.
+
+```julia
+const scoped_val = ScopedValue(1)
+const scoped_val2 = ScopedValue(2)
+
+# Enter a new dynamic scope and set value
+scoped(scoped_val => 2) do
+    @show scoped_val[] # 2 
+    @show scoped_val2[] # 2
+    scoped(scoped_val => 3, scoped_val2 => 5) do
+        @show scoped_val[] # 3 
+        @show scoped_val2[] # 5
+    end 
```

Comment:
What happens after this inner `scoped` block returns? I assume both values revert to 2, but it's not clear from this example that they do.